resources/views/site/help/import.blade.php

```diff
@@ -0,0 +1,94 @@
+@extends('site.help.partial.template', ['breadcrumb'=>'Import'])
+
+@section('section')
+
+    <div class="title">
+        <h3 class="font-weight-bold">Import</h3>
+    </div>
+    <hr>
+    <p class="lead py-3">With the Import from Instagram feature, you can seamlessly transfer your photos, captions, and even hashtags from your Instagram account to Pixelfed, ensuring a smooth transition without losing your cherished memories or creative expressions.</p>
+
+    <hr class="mb-4" />
+
+    <p class="text-center font-weight-bold">How to get your export data from Instagram:</p>
+    <ol class="pb-4">
+        <li class="mb-2">
+            <span>Follow the Instagram instructions on <strong>Downloading a copy of your data on Instagram</strong> on <a href="https://help.instagram.com/181231772500920" class="font-weight-bold">this page</a>. <strong class="text-danger small font-weight-bold">Make sure you select the JSON format</strong></span>
+        </li>
+        <li class="mb-2">
+            <span>Wait for the email from Instagram with your download link</span>
+        </li>
+        <li class="mb-2">
+            <span>Download your .zip export from Instagram</span>
+        </li>
+        <li class="mb-2">
+            <span>Navigate to the <a href="/settings/import" class="font-weight-bold">Import</a> settings page</span>
```

Comment:
I'd suggest having the link cover both words

```suggestion
            <span>Navigate to the <a href="/settings/import" class="font-weight-bold">Import settings</a> page</span>
```